devdocker

```diff
@@ -196,10 +196,26 @@ RUN mkdir -p $SRCDIR
           socket.gethostname()),
       '-e', 'DEVDOCKER_VERSION=%s' % cfg.imageVersion,
     ]
+    kDevNull = open(os.devnull, 'w')
+    if 'mutagen' in dir(cfg) and cfg.mutagen:
+      subprocess.call(
+          ['mutagen', 'project', 'terminate'],
+          stdout = kDevNull,
+          stderr = kDevNull)
+    if 'volume' in dir(cfg):
+      for volume in cfg.volume:
+        code = subprocess.call(
+            ['docker', 'volume', 'inspect', volume],
+            stdout = kDevNull,
+            stderr = kDevNull)
+        if code != 0:
+          subprocess.check_call(['docker', 'volume', 'create', volume])
     if 'environment' in dir(cfg):
       for var, value in cfg.environment.items():
         envFlags.extend(['-e', '%s=%s' % (var, value % substitutions)])
-    mountFlags = [ '-v', '%s:%s:delegated' % (srcdir, srcdir) ]
+    mountFlags = []
+    if not 'mountSrcDir' in dir(cfg) or cfg.mountSrcDir:
```

Comment:
Shall we add a comment here to not that the user may set this flag to not add the source directory via mount command and use volumes instead?

Reply:
Missed this while merging the PR. Have submitted that as a follow up change.